src/app/App.js

```diff
@@ -26,6 +27,19 @@ export const App = ({ options }) => {
   });
   const mainCointainer = document.getElementById('mainContainer');
   renderRedButton(mainCointainer);
+
+  // Modal window
+  const modalWindow = new ModalWindow();
+
+  const testButtonForWindow = document.getElementById('btnForModalWindow');
+
+  testButtonForWindow.onclick = function () {
+    const testContent = document.createElement('div');
+    testContent.innerText = 'dsadasdsadas';
+    modalWindow.show(testContent);
```

Comment:
<important>
Dlaczego jest to w App.js, a nie w osobnej funkcji???
Dlaczego nie renderujesz przycisku w App.js tylko jest w index.html?
Jeśli jest w tym coś czego nie rozumiem to napisz mi prosze tutaj. 
Ale, jeśli rozumiem to, to powinieneś ten przycisk renderować w App.js i wtedy trzeba to poprawić!

Reply:
Tak jak wspomniałem... to tylko testowo :-)